Library/ViewModels/ShippingRuleCellViewModel.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol ShippingRuleCellViewModelInputs {
+  func configureWith(_ values: ShippingRuleData)
```

Comment:
Should this be `configureWith(_ value:)` since usually we reserve the plural for arrays?